docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,244 @@
+to `/opt/machine-id`.
+
+In a production environment you will want to run Machine ID in the background
+using a service manager like systemd. However, in this guide you will run it in
+the foreground to better understand how it works.
+
+<Tabs>
+  <TabItem label="Token-based Joining">
+  ```code
+  $ tbot start \
+     --data-dir=/var/lib/teleport \
+     --destination-dir=/opt/machine-id \
+     --token=00000000000000000000000000000000 \
+     --join-method=token \
+     --ca-pin=sha256:1111111111111111111111111111111111111111111111111111111111111111 \
+     --auth-server=auth.example.com:3025
+  ```
+  </TabItem>
+  <TabItem label="IAM Method">
+  ```code
+  $ tbot start \
+     --data-dir=/var/lib/teleport \
+     --destination-dir=/opt/machine-id \
+     --token=iam-token \
+     --join-method=iam \
+     --ca-pin=sha256:1111111111111111111111111111111111111111111111111111111111111111 \
+     --auth-server=auth.example.com:3025
+  ```
+  </TabItem>
+</Tabs>
+
+Replace the following fields with values from your own cluster.
+
+* `token` is the token output by the `tctl bots add` command or the name of your IAM method token
+* `ca-pin` is the CA Pin for your Teleport cluster, and is output by the `tctl bots add` command
+* `destination-dir` is where Machine ID writes the renewable certificates to, this certificate is only used by Machine ID and should not be used by applications and tools
+* `data-dir` is where Machine ID writes the short-lived certificate. This certificate should be used by applications and tools.
+<Details scope={["cloud"]} scopeOnly={true}>
+* `auth-server` is the address of your Teleport Auth Server
+</Details>
+<Details scope={["oss","enterprise"]} scopeOnly={true}>
+* `auth-server` is the address of your Teleport Cloud Proxy Server
+</Details>
+
+Now that Machine ID has successfully started, let's investigate the
+`/opt/machine-id` directory to see what was written to disk.
+
+```
+$ tree /opt/machine-id
+machine-id
+├── key
+├── key.pub
+├── known_hosts
+├── ssh_config
+├── sshcacerts
+├── sshcert
+├── tlscacerts
+└── tlscert
+
+0 directories, 8 files
+```
+
+This directory contains private key material in the `key.*` files, SSH
+certificates in the `ssh*` files, X.509 certificates in the `tls*` files, and
+OpenSSH configuration in the `ssh_config` and `known_hosts` files to make it easy
+to integrate Machine ID with external applications and tools.
+
+## Step 4/4. Use certificates issued by Machine ID
+
+To use Machine ID, find a host that you want to connect to within your cluster
+using `tsh ls`. You might see output like the following on your system.
+
+```code
+$ tsh ls
+Node Name Address        Labels                        
+--------- -------------- ----------------------------- 
+node-name  127.0.0.1:3022 arch=x86_64,group=api-servers 
+```
+
+To use Machine ID with the OpenSSH integration, run the following command to
+connect to `node-name`.
+
+```
+ssh -F /opt/machine-id/ssh_config root@node-name
```

Comment:
We should add a note, that this requires port 22 to be open to the machine. In my case, I only had port 22 open for my home IP and I had to open it up.

Reply:
Oh, I think this might be a typo in the docs, the node name needs to be suffixed with the cluster name, e.g.

```
ssh -F /opt/machine-id/ssh_config root@node-name.example.com
```

(It uses the same logic as `tsh config`)

Reply:
That did the trick, I would suggest we make it clear that `example.com` is the cluster name, as it's easy to get confused with Top Level Domains